python/cuml/manifold/t_sne.pyx

```diff
@@ -436,24 +429,25 @@ class TSNE(Base):
                 Embedding of the training data in low-dimensional space.
         """
         self.fit(X, convert_dtype=convert_dtype)
+        out_type = self._get_output_type(X)
 
         if isinstance(X, cudf.DataFrame):
-            if isinstance(self.embedding_, cudf.DataFrame):
-                return self.embedding_
+            if isinstance(self._embedding_, cudf.DataFrame):
```

Comment:
all of these `if` type checks can be removed and reduced to using `self._embedding_.to_output(out_type)` like it is being done below